cmd/shell.go

```diff
@@ -178,6 +180,11 @@ type Config struct {
 
 		DNSResolver string
 		DNSTimeout  string
+
```

Comment:
doop, this was just to fix to the external cert importer errors, didn't mean to push, not sure when this got broken :/